components/bounty/comments/comment/view.tsx

```diff
@@ -30,15 +28,15 @@ export default function Comment({
         <div className="d-flex align-items-baseline flex-wrap">
           <FlexColumn className="justify-content-center">
             <AvatarOrIdenticon
-              user={user?.githubLogin}
+              user={user?.handle}
```

Comment:
We no longer need to pass the user. Why don't we get the avatar from github anymore.